tools/ci_build/github/azure-pipelines/nuget/templates/gpu.yml

```diff
@@ -220,6 +220,8 @@ jobs:
              mkdir %%~ni\runtimes\linux-x64
              mkdir %%~ni\runtimes\linux-x64\native
              move linux-x64\linux-x64\libonnxruntime.so %%~ni\runtimes\linux-x64\native\libonnxruntime.so
+             move linux-x64\linux-x64\libonnxruntime_providers_shared.so %%~ni\runtimes\linux-x64\native\libonnxruntime_providers_shared.so
```

Comment:
if we could have done a wildcard like move libonnxruntime* , it could have covered all the shared libs and other ones (e.g. TensorRT) for the future?

Reply:
Should I switch it to *.so?

Reply:
Thought about it a bit more, will fix the immediate issue this way, and wait on the *.so since that could potentially break things as easily as fix things and it'd be worse to accidentally start including extra files than missing one.